client/components/Mentions.vue

```diff
@@ -37,11 +37,14 @@
 							</span>
 						</div>
 						<div>
-							<span class="close-tooltip tooltipped tooltipped-w" aria-label="Close">
+							<span
+								class="close-tooltip tooltipped tooltipped-w"
+								aria-label="Dismiss this mention"
```

Comment:
Seems to be a nested aria-label here, where the outer one has precedence in the UI (tooltip). Figured it'd make more sense to change this to be the same as the button's aria-label?